packages/connect/src/data/config.ts

```diff
@@ -259,5 +259,9 @@ export const config = {
                 T2B1: '2.7.0',
             },
         },
+        {
+            capabilities: ['entropyCheck'],
+            min: { T1B1: '0', T2T1: '2.8.7', T2B1: '2.8.7' },
```

Comment:
`T3T1` missing intentionally?

Reply:
not intentionally, i just forgot about them :)

Reply:
done